_posts/2018-11-12-sr2019-rules-released.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: news
+title: SR2019 Rules Published & Useful Resources
+---
+
+We had a successful Kickstart on Saturday and it was great to see the excited teams getting familiar with our kit.
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the other robots within the zone boundaries. ([full rules][rules])
+
+We’ve already seen the teams come up with some innovative solutions and it’s going to be interesting to see how they develop.
+
+All competitors should now have user accounts for our website. This includes our [IDE][ide] for teams to write their code in. This also includes [our forums][forums] which are a great place to brainstorm with other teams and discuss any problems which you run into.
+
+(See [our docs][account-info] if you are not able to log in.)
+
+If you weren’t able to attend Kickstart this year, or would like a recap, you can download:
+- [The presentation][presentation] in which we explain this year’s game and how the competition is running this year.
+- [The microgames][microgames] which are small activities designed to let you become familiar with the Student Robotics kit.
+
+[rules]: {{ site.baseurl }}/docs/rules/
+[ide]: {{ site.baseurl }}/ide/
+[account-info]: {{ site.baseurl }}/docs/team_admin/user_accounts/
+[forums]: {{ site.baseurl }}/forums/
+[presentation]: {{ site.baseurl }}/resources/sr2019/kickstart-keynote-sr2019.pdf
+[microgames]: {{ site.baseurl }}/resources/sr2019/microgames-sr2019.pdf
```

Comment:
This document contains a reference to the tablets - we really ought to fix that rather than publish a knowingly-wrong document.

Reply:
I've fixed the Google Doc in this regard.

Reply:
@antoinepetty I've just spotted that another of the pastebin links was to a previous' year. I've updated that too now.

Reply:
Thanks, I have updated the PDF now